src/features/singleRight/delegate/ChooseServicePage/ChooseServicePage.module.css

```diff
@@ -85,3 +85,11 @@
     width: 100%;
   }
 }
+
+.pageContent {
+  min-height: 1200px;
```

Comment:
I understand why you've done this, but right now, it makes the page look a bit strange if there are no results to display.
Maybe pulling the buttons down to the lower right corner could help?

![image](https://github.com/Altinn/altinn-access-management-frontend/assets/22012997/44755b4d-8969-4a88-987b-689b7c930f50)

Reply:
Fixed it now but on small screens I still think it should stay close to the ActionBars